spring-session/src/main/java/org/springframework/session/MapSession.java

```diff
@@ -40,7 +40,7 @@
  * @since 1.0
  * @author Rob Winch
  */
-public final class MapSession implements ExpiringSession, Serializable {
+public class MapSession implements ExpiringSession, Serializable {
```

Comment:
I'm rather wary of it at the moment. FST's own dependencies are pretty minimal (Jackson, Objensis, and Javassist), but minor updates are released very frequently (20 in the last 12 months) and that may prove problematic in the future.

If it were me, I'd be looking for alternatives that are less of a moving target. If there aren't any, I'd consider using jarjar and embedding FST so that you can have complete control over the version that's used.